README.md

```diff
@@ -2,15 +2,15 @@
 
 Web Application where people new to Machine Learning can input a dataset and experiment around with basic Pytorch modules through a drag and drop interface
 
-> **Deployed website:** https://datasciencegt-dlp.com/ </br> > **GitHub repo:** https://github.com/DSGT-DLP/Deep-Learning-Playground
+> **Deployed website:** https://datasciencegt-dlp.com </br> **GitHub repo:** https://github.com/DSGT-DLP/Deep-Learning-Playground
 
 # Getting Started
 
 ### Prerequisites
 
 Have the following installed first:
 
-1. [Node.js v18 via NVM](https://github.com/nvm-sh/nvm#installing-and-updating) (Install nvm first, and then install node & npm using nvm)
+1. [Node.js v20 via NVM](https://github.com/nvm-sh/nvm#installing-and-updating) (Install nvm first, and then install node & npm using nvm)
 1. [Yarn 1.x](https://classic.yarnpkg.com/lang/en/docs/install) (Must be installed after npm. May upgrade to Yarn Modern in the future, keep an eye out for that!)
```

Comment:
do we need people to install yarn even though we are on pnpm?

Reply:
Good point. Removed